src/main/python/main/ayab/engine/engine.py

```diff
@@ -70,7 +70,7 @@ def __init__(self, parent: GuiMain):
         self.control = Control(parent, self)
         self.__feedback = FeedbackHandler(parent)
         self.__logger = logging.getLogger(type(self).__name__)
-        self.setWindowTitle("Machine: " + Machine(self.config.machine).name)
+        self.refresh_settings_change()
```

Comment:
This causes `self.config.refresh()` to be called twice in `__init__`, maybe this could be improved?